docs/ARCHITECTURE_DOCUMENT.md

```diff
@@ -0,0 +1,235 @@
+|  ---  |  ---  |  ---  |  ---  |
+| cpf | string[25] | obrigatório, único, validado | CPF do parente |
+
+### PATIENT:
+
+| Atributo | Tipo | Característica | Descrição |
+|  ---  |  ---  |  ---  |  ---  |
+| urgency | integer | obrigatório | Grau de urgência para atendimento do paciente |
+| ses | integer | obrigatório, único, validado | Número SES do paciente |
+
+### DOCTOR:
+
+| Atributo | Tipo | Característica| Descrição |
+|----|------|-------|--------|
+| CRM | Integer | Obrigatório, único | Número CRM do doutor |
+| specialty | String[20] | Obrigatório | Especialidade |
+| CPF | String[25] | Obrigatório, único, validado | CPF do doutor |
+
+### POST:
+
+| Atributo | Tipo | Característica | Descrição |
+|--|--|--|--|
+| title |string[100] | obrigatório | Título do post |
+| description | text | obrigatório | Descrição do post |
+| author | user | obrigatório | Nome do autor do post |
+| created_at | date | automático | Data de criação do post |
+| _at | date | automático | Data de modificação do post |
+| updated_by | user | obrigatório | Usuário que modificou o post |
+|slug | SlugField | obrigatório | Usado para inserir URLs nomeadas |
+
+### MEDICAL QUESTIONARY:
+
+| Atributo | Tipo | Característica | Descrição |
+|--|---|--|--|
+| psychosocial_risk | integer | obrigatório | Risco psicossocial
+| health_risk |integer | obrigatório | Risco de vida |
+| family_risk | integer | obrigatorio | Risco familiar |
+| total_risk | integer | opcional | Risco total |
+
+### QUEUE:
+
+| Atributo | Tipo | Característica | Descrição |
+|--|----|----|--|
+| speciality | string[50] | obrigatório | Especialidade |
+| time_left | dateTimeField | automático | Tempo faltando |
+| position | integer | automático | Posição |
+
+### EVENTS:
+
+| Atributo | Tipo | Característica| Descrição |
+|---|----|---|---|
+| name | string[100] | obrigatório |Nome do evento |
+| date | dateTime | obrigatório | Data do evento |
+| address | string[60] | obrigatório | Endereço do evento |
+| description | text | obrigatório | Descrição do evento |
+
+### APPOINTMENTS:
+
+| Atributo | Tipo | Característica | Descrição |
+|  ---  |  ---  |  ---  |  ---  |
+| name | string[100] | obrigatório | Nome do compromisso |
+| date | date/time | obrigatório | Data do compromisso |
+| description | text | opcional | Descrição do compromisso |
+
+### TOPIC:
+
+| Atributo | Tipo | Característica | Descrição |
+|  ---  |  ---  |  ---  |  ---  |
+| title | string[100] | obrigatório | Título do tópico |
+| subject | text | obrigatório | Assunto do tópico |
+| last_update | date/time | automático | Último data atualizada |
+| slug | SlugField | string | Usado para inserir URLs renomeadas |
+
+### RELACIONAMENTOS:
+
+#### 1 - APPOINTMENTS tem USERS (Doctor)
+
+Um doutor pode ter uma ou várias consultas, mas toda consulta tem um médico.
+Cardinalidade: 1 X N
+
+#### 2 - APPOINTMENTS tem USERS (pacientes)
+
+Um paciente pode ter várias consultas, mas toda consulta tem um paciente.
```

Comment:
Eu já vi consultas que tem vários pacientes.

Se não for o caso utilize o padrão que falei acima:

Um paciente pode ter uma ou várias consultas, mas uma consulta pertence a um único paciente.